### PR TITLE
Use new issue template

### PR DESCRIPTION
**Pull request by BastianZim**
Like in https://github.com/conda-forge/staged-recipes/pull/17366 this brings the new [issue template format](https://gh-community.github.io/issue-template-feedback/structured/) to this repo. The template is in beta but GitHub is using it itself in their [docs](https://github.com/github/docs/tree/main/.github/ISSUE_TEMPLATE) so I would consider it stable enough.

Please feel free to try it out for yourself in my fork.

Two sample issues are available here:
* https://github.com/BastianZim/conda-forge.github.io/issues/5
* https://github.com/BastianZim/conda-forge.github.io/issues/6

### CODE REVIEW

**Comment by BastianZim** [comment]
I'm not sure where the old template is from as I couldn't find it in this repo. Is this set somewhere for the whole organisation?

**Comment by BastianZim** [comment]
@jaimergp Have added it here now as discussed in https://github.com/conda-forge/staged-recipes/pull/17366

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
@conda-forge/core , any other thoughts here? 🙂

**Comment by jakirkham** [comment]
Also @BastianZim would it be possible to do this for [the `.github` org repo]( https://github.com/conda-forge/.github/tree/master/.github )? That way we could have similar changes for the feedstocks 😄

**Comment by BastianZim** [comment]
Sure! I’m currently travelling with satellite internet but will do that when back.

**Comment by beckermr** [comment]
No rush. We likely won't merge right away.

**Comment by BastianZim** [comment]
PR is in https://github.com/conda-forge/.github/pull/9

File: .github/ISSUE_TEMPLATE/error_template.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Report an issue
+description: Report an issue you are experiencing.
+labels: General
```
Comment (BastianZim) [inline .github/ISSUE_TEMPLATE/error_template.yml:3]: @beckermr I just realised I used the same label here as for the general issue – will fix that. What kind of labels are available in this repo to use?

Comment (jakirkham) [reply .github/ISSUE_TEMPLATE/error_template.yml:3]: Here is [the list of labels]( https://github.com/conda-forge/conda-forge.github.io/labels ). That said, I doubt we want to use all of those, but some reasonable subset might make sense

Comment (BastianZim) [reply .github/ISSUE_TEMPLATE/error_template.yml:3]: Ahh thanks! Didn't know you could see them not being a member.

PR is in https://github.com/conda-forge/conda-forge.github.io/pull/1594

**Pull request merged**